build.gradle

```diff
@@ -3,7 +3,6 @@ subprojects {
     apply plugin: 'osgi'
     apply plugin: 'eclipse'
     
-    version = '1.1.0-SNAPSHOT'
```

Comment:
Is there a specific reason for this commit? Please elaborate.

Reply:
For convenience, I wanted to override versioning from the command line. E.g.:

`sh "./gradlew -Pversion=$VersionTag citeproc-java:jar"`

where `$VersionTag` includes the git branch number and the build number.

I'm using Jenkins internally to build the project. My versioning includes the BUILD_NUMBER. I could overwrite the version number in the `build.gradle` every build, but I prefer to set the versioning in the command line. This leads to less commits and is less error prone. The build still reads the version info from gradle.properties, so:
- for local builds (no `version` override) I get snapshots,
- and for jenkins builds, I get git branch versioning plus build info.

The easiest way I found to achieve this was extracting the versioning info out of the `build.gradle` into the `gradle.properties`. (See https://stackoverflow.com/a/40239092/413020.)

Reply:
I have created https://github.com/michel-kraemer/citeproc-java/pull/68